packages/webapp/src/containers/Filter/Animals/index.tsx

```diff
@@ -0,0 +1,183 @@
+/*
+ *  Copyright 2024 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+import { useSelector } from 'react-redux';
+import PropTypes from 'prop-types';
+import { useTranslation } from 'react-i18next';
+import FilterGroup from '../../../components/Filter/FilterGroup';
+import type { ReduxFilterEntity, ContainerOnChangeCallback, Location } from '../types';
+import { FilterType, type ComponentFilter } from '../../../components/Filter/types';
+import {
+  useGetDefaultAnimalTypesQuery,
+  useGetCustomAnimalTypesQuery,
+  useGetDefaultAnimalBreedsQuery,
+  useGetCustomAnimalBreedsQuery,
+  useGetAnimalGroupsQuery,
+  useGetAnimalSexesQuery,
+} from '../../../store/api/apiSlice';
+import { generateUniqueAnimalId } from '../../../util/animal';
+import { useVisibleBreeds } from './useVisibleBreeds';
+import { locationsSelector } from '../../locationSlice';
+import { AnimalOrBatchKeys, AnimalsFilterKeys } from './types';
+import { sortFilterOptions } from '../../../components/Filter/utils';
+
+interface AnimalsFilterContentProps {
+  animalsFilter: ReduxFilterEntity<AnimalsFilterKeys>;
+  filterRef: React.RefObject<ReduxFilterEntity<AnimalsFilterKeys>>;
+  filterContainerClassName?: string;
+  onChange: ContainerOnChangeCallback;
+}
+
+const AnimalsFilterContent = ({
+  animalsFilter,
+  filterRef,
+  filterContainerClassName,
+  onChange,
+}: AnimalsFilterContentProps) => {
+  const { t } = useTranslation(['translation', 'filter']);
+
+  const locations: Location[] = useSelector(locationsSelector);
+
+  const { data: defaultTypes = [] } = useGetDefaultAnimalTypesQuery();
+  const { data: customTypes = [] } = useGetCustomAnimalTypesQuery();
```

Comment:
I was also unsure about this, because that's exactly how I initially interpreted Loïc's `ON YOUR FARM` Figma note. But I brought this question to tech daily earlier this week and we decided it actually just mean types _available_ to your farm (default + your own custom) rather than types _active_ on your farm since that is more true to the existing filter pattern.

And then as  a bonus it's much easier to implement since the note is also applied to breed, and we have no `count` setup on breed at all!

Reply:
Sorry I can't remember the discussion...
I thought it would be nice if breeds of types that I don't have in my farm were hidden, but I understand!